src/content/okteto-license.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Okteto License
+description: This page provides information about how Okteto's license works for the products we provide.
+sidebar_label: Okteto License
+id: okteto-license
+---
+
+Okteto requires a license key to use any of our paid products. This document explains how our license works for the different products we provide to remove any uncertainty about feature or usage limitations during a trial period or for your chosen tier of paid service.
+
+# Why a license is required
+
+We require you obtain a license to use Okteto because the product is highly customizable and we want to ensure you have a great experience getting it installed and configured. Obtaining a license from us kicks off specific customer success workflows to help us provide you with the right guidance and support.
+
+Don't worry. We don't require you speak with someone from our Sales team to try Okteto or deal with more friction than is necessary. We want you to experience Okteto in the ways we believe will inform your decisions and requiring a license is the first step on that journey.
```

Comment:
can we also include that you don't need a credit card?

Reply:
```suggestion
Don't worry. We don't require you speak with someone from our Sales team to try Okteto or deal with more friction than is necessary. There's also no requirement to provide us with credit card information. We want you to experience Okteto in the ways we believe will inform your decisions and requiring a license is the first step on that journey.
```

Would this work?

Reply:
yep!